src/js/core/methods/EthereumSignTypedData.js

```diff
@@ -84,9 +93,7 @@ export default class EthereumSignTypedData extends AbstractMethod<'ethereumSignT
             };
         }
 
-        validateParams(this.params, [{ name: 'data', type: 'object', required: true }]);
```

Comment:
No longer needed in  the `run()` command, since these Trezor T parameters are now always required, and so this is validated in the `init()` function.